viewer/src/data/logTests/itars/all.json

```diff
@@ -3383,9 +3443,13 @@
     "round": 3,
     "turn": 0,
     "changes": [
+      {
+        "source": "beginGame",
+        "changes": "1o, 1k, 1t"
+      },
       {
         "source": "income",
-        "changes": "1o, 4k, 2t, 4pw"
+        "changes": "1t, 3k, 4pw"
```

Comment:
Maybe `"source": "beginGame"` should only apply to the non-income events. Or be called `'factionBoard'`?

It's weird seeing `beginGame` at round 3, ...

Reply:
fixed in next pr